Rubberduck.Core/UI/AddRemoveReferences/AddRemoveReferencesViewModel.cs

```diff
@@ -25,6 +26,51 @@ public enum ReferenceFilter
 
     public class AddRemoveReferencesViewModel : ViewModelBase
     {
+        private static readonly Dictionary<string, string[]> HostFileFilters = new Dictionary<string, string[]>
+        {
+            { "EXCEL.EXE", new [] {"xlsm","xlam","xls","xla"} },
```

Comment:
I wonder if that data should belong to the various implementations of `IHostApplication`.

Reply:
I considered it, but injecting an IHostApplication will drag a SCW into the view model.

Reply:
I think that's a great idea. Want to open an issue so we don't forget?